tests/visual/assets/styles.css

```diff
@@ -27,8 +27,13 @@ body {
     margin: 10px;
 }
 
+/* Prevent animations */
 .k-no-animations,
-.k-no-animations * {
+.k-no-animations *,
+.k-i-loading:before,
+.k-i-loading:after,
+.k-loading-image::before,
+.k-loading-image::after  {
     animation: none !important;
```

Comment:
Does this need to be explicitly set with !important?

Reply:
shouldn't we add _// sass-lint:disable-line no-important_ after the !important?

Reply:
that's in the styles of the testing application:
* it's `!important` to ensure all animations of loading are disabled, otherwise the tests will go on forever
* since it's in the styles of the testing application, no linting is required :)